cmd/uplink/cmd/cp.go

```diff
@@ -84,20 +82,30 @@ func upload(ctx context.Context, src fpath.FPath, dst fpath.FPath, showProgress
 		return fmt.Errorf("source cannot be a directory: %s", src)
 	}
 
-	metainfo, streams, err := cfg.Metainfo(ctx)
+	project, err := cfg.GetProject(ctx)
 	if err != nil {
 		return err
 	}
+	defer func() {
+		err = project.Close()
+		if err != nil {
+			fmt.Printf("Error closing project uplink: %+v\n", err)
```

Comment:
What is "project uplink"? Shouldn't the error be just "Error closing project: %+v\n"?

Reply:
done